app/src/main/java/com/nextcloud/talk/data/source/local/Migrations.kt

```diff
@@ -124,4 +131,29 @@ object Migrations {
         // Change the table name to the correct one
         db.execSQL("ALTER TABLE ArbitraryStorage_dualPK RENAME TO ArbitraryStorage")
     }
+
+    fun migrateToTriplePrimaryKeyArbitraryStorage(db: SupportSQLiteDatabase) {
+        db.execSQL(
+            "CREATE TABLE ArbitraryStorage_triplePK (" +
```

Comment:
why is the order now changed to:
accountIdentifier
value
\"key\"
object

it makes more sense to use this order? (so the primary key fields come first):
accountIdentifier
\"key\"
object
value